tests/integration/BUILD

```diff
@@ -0,0 +1,53 @@
+#
```

Comment:
This is the new BUILD file that was introduced specifically for integration tests